src/LuaLib.ts

```diff
@@ -46,8 +48,10 @@ export enum LuaLibFeature {
     Iterator = "Iterator",
     LuaIteratorSpread = "LuaIteratorSpread",
     Map = "Map",
+    Match = "Match",
```

Comment:
Since this and CountVarargs are only used by the lualib internally and not by the transpiler, could we leave them out of this enum? Or would that somehow mess up the lualib compilation?

Reply:
Currently, any file in lualib/ [has to be named](https://github.com/YoRyan/TypeScriptToLua/blob/471554074a6defbb96fb27011a9c28e508e34e34/src/lualib-build/plugin.ts#L78) in this enum or the compilation will fail. The lualib builder strips the import statements from every file and constructs its own dependency graph.